terraform/iam.tf

```diff
@@ -34,11 +38,11 @@ data "aws_iam_policy_document" "lambda_policy" {
       "arn:aws:secretsmanager:${local.aws_region_name}:${local.aws_account_id}:secret:*/ct-access-token-*",
     ]
 
-    condition {
-      test     = "ArnEquals"
-      variable = "secretsmanager:resource/AllowRotationLambdaArn"
-      values   = [aws_lambda_function.commercetools_token_refresher.arn]
-    }
+    # condition {
```

Comment:
Why don't we need this one anymore? We can remove the commented code if not needed.

Reply:
hm removed it because a cyclic dependency. Let me double check